.github/workflows/backend.yaml

```diff
@@ -8,51 +8,24 @@ on:
     branches:
       - main
 
-defaults:
-  run:
-    working-directory: ./backend
-
 jobs:
   lint:
     name: "Lint"
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
 
-      - name: Install poetry
-        run: pipx install poetry
-
-      - uses: actions/setup-python@v4
-        with:
-          python-version: 3.8
-          cache: "poetry"
-
-      - name: Install dependencies
-        run: poetry install
-      - name: Lint the Code
-        run: |
-          poetry run black --check .
-          poetry run flake8
+      - name: Lint the backend code
+        run: make lint-backend
```

Comment:
No frontend linting / tests?

Reply:
There is a seperate file for that (frontend.yml)

Reply:
Ah, in this case, nevermind.